VRCBilliards/vrcbce
# The New Physics System Backport

This work was on the 1.6 experimental repo for a while.

I'm backporting it back into the main repo, as it seems to be broadly stable and it's a major feature win for the formal 1.6 update when it happens.

TO TEST:

0. Do all the prefabs pass a smoke check?
1. In desktop, use UP and DOWN to tilt the cue (locally! - no cue location replication, sadly) - does this work properly?
2. Do 9-ball and the two 4-ball carom games work?

We need to iterate on this feature a bit; I'll also be documenting and cleaning up the codebase as I go to make things spic-and-span.
